nvbench/json_printer.cu

```diff
@@ -16,24 +16,18 @@
  *  limitations under the License.
  */
 
-#include <nvbench/json_printer.cuh>
-
 #include <nvbench/axes_metadata.cuh>
 #include <nvbench/benchmark_base.cuh>
 #include <nvbench/config.cuh>
+#include <nvbench/detail/throw.cuh>
```

Comment:
Let's avoid reordering the headers in this branch and just fix the clang bug

Reply:
Reverted.